### PR TITLE
Project setup instructions

### PR DESCRIPTION
**Pull request by gsundeep-tech**
Description:
Instructions to run the application locally

### CODE REVIEW

**Comment by gsundeep-tech** [comment]
@vigneshgowrabathuni please add the front end setup instructions to the readme file.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
-# split-expense
+# About
+
 A website to split expenses based on invoice
+
+# Local setup
+
+Application is built using python & flask-restplus as the backend and react for the front end. following are the steps required to setup the application in local computer
```
Comment (vigneshgowrabathuni) [inline README.md]: Application is built using **[Python](https://www.python.org/)** and **[Flask-RESRPlus](https://flask-restplus.readthedocs.io/en/stable/)** as the backend and **[React JS](https://reactjs.org/)** as a front end technologies. Please follow the below steps for local application setup.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+
+## Backend setup
+
+Backend application uses postgres as the dependency, follow the steps to install the postgres through docker
```
Comment (vigneshgowrabathuni) [inline README.md]: - It's will be nice if we add links to the **technical terms**. 
- Each bullet point statement should start with the **Uppercase letter**.

**Review by vigneshgowrabathuni — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gsundeep-tech) [reply README.md]: Thanks for the review, Will update the requested changes :)

File: README.md (hunk 2 above)
Comment (gsundeep-tech) [reply README.md]: Thanks for the review, will update the requested changes :)

**Review by vigneshgowrabathuni — APPROVED** [review]
Back-end setup doc looks good.

**Pull request merged**